src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -772,4 +769,28 @@ private AnomalyDetectorSettings() {}
             Setting.Property.NodeScope,
             Setting.Property.Dynamic
         );
+
+    // ======================================
+    // stats/profile API setting
+    // ======================================
+    // the max number of models to return per node.
+    // the setting is used to limit resource usage due to showing models
+    public static final Setting<Integer> MAX_MODEL_SZIE = Setting
```

Comment:
Typo in setting name

Reply:
Could you be more explicit about the typo? Didn't find it.

Reply:
`SZIE` => `SIZE`

Reply:
fixed